wagtail/search/tests/test_backends.py

```diff
@@ -507,6 +516,17 @@ def test_operators_combination(self):
                              'The Rust Programming Language',
                              'Two Scoops of Django 1.11'})
 
+    def test_prefix_single_word(self):
+        results = self.backend.search(Prefix('pro'), models.Book.objects.all())
+        self.assertSetEqual({r.title for r in results},
+                            {'The Rust Programming Language'})
+
+    def test_prefix_multiple_words(self):
+        results = self.backend.search(Prefix('rust pro'),
```

Comment:
"prefix" is a single term query on Elasticsearch, so this example won't return anything there

Reply:
According to [this article](https://qbox.io/blog/elasticsearch-queries-prefix-term-multimatch-bool), prefix queries in Elasticsearch can contain multiple terms.

Reply:
Nice, makes things a lot easier!

Reply:
I’m not sure what you mean by this. Anyway, the test is marked as an expected failure on Elasticsearch.